pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -48,6 +48,35 @@ let
       };
       version = "5.212.0-alpha4";
     };
+
+    # Even if developed in the public, QtWebEngine does not have official
+    # releases or new tags since the Qt company made 5.15.3 proprietary.
+    # Apparently they care more about licensing than the security of their users.
+    # See https://lists.qt-project.org/pipermail/interest/2021-March/036387.html
+    qtwebengine =
+      let
+        branchName = "5.15.3";
+        rev = "a059e7404a6db799f4da0ad696e65ae9c854b4b0";
+      in
+      {
+        version = "${branchName}-${lib.substring 0 8 rev}";
+
+        src = fetchgit {
+          url = "https://github.com/qt/qtwebengine.git";
+          sha256 = "19l1i4sk3pvnwbvz5s97jchqawfz8k1xmjza29bgvkp1zz96r0jx";
+          inherit rev branchName;
+          leaveDotGit = true;
+        };
+
+        postUnpack = ''
+          # Set Chromium branch
+          git submodule set-branch --branch 87-based src/3rdparty
```

Comment:
Wouldn't an update to that branch make us have to change the hash?

Reply:
I was wondering about this as well. Are we sure it will always give the same result?